spec/ddoc.dd

```diff
@@ -542,12 +546,23 @@ Identifiers in documentation comments that are function parameters or are
 names that are in scope at the associated declaration are emphasized in
 the output.
 This emphasis can take the form of italics, boldface, a hyperlink, etc.
-How it is emphasized depends on what it is - a function parameter, type,
+How it is emphasized depends on what it is $(MDASH) a function parameter, type,
 D keyword, etc.
 To prevent unintended emphasis of an identifier, it can be preceded by
 an underscore ($(UNDERSCORE)). The underscore will be stripped from the output.
 )
 
+$(H3 $(LNAME2 urls, URLs))
+
+$(P
+URLs are sequences of characters starting with 'http://' or 'https://',
+continue with one or more characters from the set of letters, digits and
+`-_?=%&/+#~.`, and contain at least one `.`.
```

Comment:
This looks very weird in the resulting page. I think it'd be better to spell it out and write `...and contain at least one period.`